src/super_gradients/training/transforms/all_transforms.py

```diff
@@ -118,3 +120,6 @@
     Transforms.RandomAutocontrast: RandomAutocontrast,
     Transforms.RandomEqualize: RandomEqualize,
 }
+
+ALBUMENTATIONS_TRANSFORMS = {name: cls for name, cls in inspect.getmembers(importlib.import_module("albumentations"), inspect.isclass)}
+ALBUMENTATIONS_COMP_TRANSFORMS = {name: cls for name, cls in inspect.getmembers(importlib.import_module("albumentations.core.composition"), inspect.isclass)}
```

Comment:
inspect.isclass & issubclass(A.BasicCompose) ?

Reply:
Good point, added in 7ec67fb008f8880c4b75348f10d865cfa5ede9b1